File: uva11479.cpp
#include<stdio.h>
int main()
{
    long long t,k,ara[4],b,c,i,j;
    scanf("%lld",&t);
    for(k=1;k<=t;k++)
    {
        scanf("%lld%lld%lld",&ara[0],&ara[1],&ara[2]);


            for(i=0;i<2;i++)
                for(j=i+1;j<3;j++)
                if(ara[i]>ara[j])
            {
                ara[i]^=ara[j];
                ara[j]^=ara[i];
                ara[i]^=ara[j];
            }
            if(ara[2]>=ara[0]+ara[1]||ara[0]<=0)
                printf("Case %lld: Invalid\n",k);

           else if(ara[0]==ara[1]&&ara[0]==ara[2])printf("Case %lld: Equilateral\n",k);

            else if(ara[0]==ara[1]||ara[1]==ara[2])
                 printf("Case %lld: Isosceles\n",k);

            else printf("Case %lld: Scalene\n",k);

    }
}

File: UVa easy problems solved in Java/uva_11479.java
import java.util.Scanner;
public class Main
{

    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        //String s;
        long t,a,b,c,i;
    while(sc.hasNext())
    {
        t = sc.nextLong();
        i=1;
        while(i<=t)
        {
            a = sc.nextLong();
            b = sc.nextLong();
            c = sc.nextLong();
            if((a+b)<=c || (b+c)<=a || (c+a)<=b)
                System.out.printf("Case %d: Invalid\n",i);
            else if(a<=0 || b<=0 || c<=0)
                System.out.printf("Case %d: Invalid\n",i);
            else if(a==b && b==c)
                System.out.printf("Case %d: Equilateral\n",i);
            else if(a==b || b==c || c==a)
                System.out.printf("Case %d: Isosceles\n",i);
            else
                System.out.printf("Case %d: Scalene\n",i);
            i++;
        }

    }
    }
}
